src/client.h

```diff
@@ -554,6 +554,8 @@ class Client : public con::PeerHandler, public InventoryManager, public IGameDef
 	// And relations to objects
 	std::unordered_map<int, u16> m_sounds_to_objects;
 
+
+	std::unordered_map<u32, u32> m_particles_server_to_client;
```

Comment:
the empty line should be after this def, not before

Reply:
done